MATLAB.gitlab-ci.yml

```diff
@@ -3,10 +3,11 @@
 # This specific template is located at:
 # https://gitlab.com/gitlab-org/gitlab/-/blob/master/lib/gitlab/ci/templates/MATLAB.gitlab-ci.yml
 
-# Use this template to run MATLAB and Simulink as part of your CI/CD pipeline. The template includes three jobs:
+# Use this template to run MATLAB and Simulink as part of your CI/CD pipeline. The template includes four jobs:
 # - `command`: Run MATLAB scripts, functions, and statements.
 # - `test`: Run tests authored using the MATLAB unit testing framework or Simulink Test.
 # - `test_artifacts`: Run MATLAB and Simulink tests, and generate test and coverage artifacts.
+# - `build`: Run builds using the MATLAB Build Tool.
```

Comment:
As in here, use "a build" instead of "builds" because a single invocation of buildtool results in a single build: https://github.com/matlab-actions#run-matlab-build:~:text=To%20run%20a%20MATLAB%20build